### PR TITLE
being able to print plain string message as cli output

### PR DESCRIPTION
**Pull request by honnix**

### CODE REVIEW

File: styx-cli/src/main/java/com/spotify/styx/cli/Main.java [hunk 1]
```diff
@@ -415,6 +415,9 @@ private void triggerWorkflowInstance()
     final WorkflowInstance workflowInstance = getWorkflowInstance(namespace);
     final ByteString payload = serialize(workflowInstance);
     send(Request.forUri(apiUrl("scheduler", "trigger"), "POST").withPayload(payload));
+    cliOutput.printMessage(CliOutput.CliMessage.create(
```
Comment (danielnorberg) [inline styx-cli/src/main/java/com/spotify/styx/cli/Main.java]: Would it make sense to include the workflow id and parameter in the message?

Perhaps the json output should be more structured?

Comment (honnix) [inline styx-cli/src/main/java/com/spotify/styx/cli/Main.java]: Thought about this actually. How to make it generic enough for other cases. How about:

```
{
   "type":"info/warning/error",
   "message":"...",
   "details":"..."
}
```

In this case, `details` may contain workflow information.

Comment (danielnorberg) [inline styx-cli/src/main/java/com/spotify/styx/cli/Main.java]: How about outputting the WorkflowInstance returned by the POST?

Comment (danielnorberg) [inline styx-cli/src/main/java/com/spotify/styx/cli/Main.java]: I.e.

```
{
  "workflow": {
    "component_id": "...", 
    "workflow_id"
  },
  "parameter": "..."
}
```

**Comment by danielnorberg** [comment]
👍

**Pull request merged**